src/Raven.Server/Documents/AttachmentsStorage.cs

```diff
@@ -1251,6 +1253,8 @@ private void CreateTombstone(DocumentsOperationContext context, Slice keySlice,
             if (table.VerifyKeyExists(keySlice))
                 return; // attachments (and attachment tombstones) are immutable, we can safely ignore this
 
+            var flags = artificial ? DocumentFlags.Artificial : DocumentFlags.None;
```

Comment:
this is always used with another flag that points to the source, e.g. FromIndex

Shouldn't we put here 'FromResharding' ?

Reply:
added `FromResharding` flag